constants/megaMenuLinks.ts

```diff
@@ -0,0 +1,337 @@
+type SegmentData = {
+  action: string
+}
+
+/**
+ * Represent a navigation link for the menus of the site
+ */
+interface MegaMenuLink {
+  /** The visible name of the link */
+  label: string
+  /** Where we want to go */
+  segment?: SegmentData
+  /** Only for <a> links. Use `_blank` to open the link in a new tab */
+  target?: string
+  /** Only for <a> links. Specifies the relationship between the current document
+   *  and the linked document. Possible values: `nofollow`, `noreferrer`, `noopener`,
+   *  `me` or a combination of them
+   */
+  rel?: string
+}
+
+type Links = MegaMenuLink
+
+type LinksList = {
+  links: Links[]
+}
+
+type Column = LinksList[]
+
+const MEGA_MENU_LINKS_COLUMNS: Column[] = [
```

Comment:
We should change this const name because is too much generic. Maybe we can change it with `MEGA_MENU_TEXTBOOK_LINKS`... Ideas are welcome :)